instrumentation/oshi/library/src/main/java/io/opentelemetry/instrumentation/oshi/SystemMetrics.java

```diff
@@ -32,15 +31,15 @@ public static void registerObservers() {
     meter
         .longUpDownSumObserverBuilder("system.memory.usage")
         .setDescription("System memory usage")
-        .setUnit("bytes")
+        .setUnit("By")
         .build()
         .setCallback(
             new Callback<LongResult>() {
               @Override
               public void update(LongResult r) {
                 GlobalMemory mem = hal.getMemory();
-                r.observe(mem.getTotal() - mem.getAvailable(), Labels.of(TYPE_LABEL_KEY, "used"));
-                r.observe(mem.getAvailable(), Labels.of(TYPE_LABEL_KEY, "free"));
+                r.observe(mem.getTotal() - mem.getAvailable(), Labels.of("state", "used"));
```

Comment:
Should we create constants for all the labels?

Reply:
updated for label keys